app/src/main/java/com/jnj/vaccinetracker/vaccinesoverview/screens/VaccinesOverviewFragment.kt

```diff
@@ -470,6 +473,15 @@ class VaccinesOverviewFragment : BaseFragment(),
             workbook.write(outputStream)
             workbook.close()
         }
+
+        val exportedFile = File(requireContext().getExternalFilesDir(null), fileName)
+        if (exportedFile.exists()) {
+            Log.d("Export", "File path: ${exportedFile.absolutePath}")
```

Comment:
this log can be removed. this information is useless

Reply:
Addressed comments.  cc @druchniewicz